test/functional/pdo_sqlsrv/PDO_ConnPool_Unix.phpt

```diff
@@ -1,63 +1,47 @@
 --TEST--
-PDO Connection Pooling Test on Unix
+PDO_SQLSRV Connection Pooling Test on Unix
 --DESCRIPTION--
-This test assumes odbcinst.ini has not been modified. 
-This test also requires root privileges to modify odbcinst.ini file on Linux.
+This test assumes the deafult odbcinst.ini has not been modified. 
 --SKIPIF--
 <?php if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') die("Skipped: Test for Linux and Mac"); ?>
 --FILE--
 <?php
-// On Bamboo we must use sudo to fiddle with odbcinst.ini
-// On travis-ci we can't use sudo
-$sudo = '';
-$user = posix_getpwuid(posix_geteuid());
-if (strtolower($user['name']) != 'root')
-{
-    $sudo = 'sudo ';
-}
-
 $lines_to_add="CPTimeout=5\n[ODBC]\nPooling=Yes\n";
 
-//get odbcinst.ini location
+//get default odbcinst.ini location
 $lines = explode("\n", shell_exec("odbcinst -j"));
 $odbcinst_ini = explode(" ", $lines[1])[1];
+$custom_odbcinst_ini = dirname(__FILE__)."/odbcinst.ini";
 
-//back up the odbcinst.ini file
-shell_exec($sudo."cp $odbcinst_ini $odbcinst_ini.bak");
+//copy the default odbcinst.ini into the current folder
+copy( $odbcinst_ini, $custom_odbcinst_ini);
 
 //enable pooling by modifying the odbcinst.ini file
-$current = file_get_contents($odbcinst_ini);
+$current = file_get_contents($custom_odbcinst_ini);
 $current.=$lines_to_add;
-shell_exec("cp $odbcinst_ini .");
-file_put_contents("odbcinst.ini", $current);
-shell_exec($sudo."cp odbcinst.ini $odbcinst_ini");
+file_put_contents($custom_odbcinst_ini, $current);
 
 //Creating a new php process, because for changes in odbcinst.ini file to affect pooling, drivers must be reloaded.
-print_r(shell_exec(PHP_BINARY." ".dirname(__FILE__)."/isPooled.php"));
+//Also setting the odbcini path to the current folder for the same process.
+//This will let us modify odbcinst.ini without root permissions
+print_r(shell_exec("export ODBCSYSINI=".dirname(__FILE__)."&&".PHP_BINARY." ".dirname(__FILE__)."/isPooled.php"));
+
```

Comment:
ODBCSYSINI - Overloads path to unixODBC configuration files. By default equals to '/etc'.
ODBCINSTINI - Overloads the name of the drivers configuration file. It is relative to ODBCSYSINI and by default set to 'odbcinst.ini'.